src/state/schedule-data/schedule-data.model.ts

```diff
@@ -39,6 +40,16 @@ export interface ScheduleDataModel {
   isCorrupted: boolean;
 }
 
+export interface ServerDataModel {
+  schedule_info: MonthInfoModel;
+  month_info: ServerInfoModel;
+  employee_info: WorkersInfoModel;
+  shifts: WorkerShiftsModel;
+  shift_types: ShiftTypesDict;
+  isAutoGenerated: boolean;
+  isCorrupted: boolean;
+}
```

Comment:
`ServerScheduleDataModel`
Jak wyżej.
Plus proponuję wynieść to do osobnego pliku